tests/ja4-rules-requires/test.yaml

```diff
@@ -0,0 +1,18 @@
+requires:
+  min-version: 7.0.3
+  files:
+    - rust/src/ja4.rs
+  script:
+    - grep 'ProvidesFeature(FEATURE_JA4);' src/app-layer-ssl.c > /dev/null
```

Comment:
Why not use --dump-features here ? What is the difference with the previous test ?

Reply:
In this case we want to test if the `requires` functionality works when JA4 support has been enabled at compile time.

Reply:
This looks fragile, like if I rename `ProvidesFeature` to `ProvidingFeature`, this test will get skipped...